DEVELOPMENT.md

```diff
@@ -26,6 +26,25 @@ To run end-to-end tests you will also need:
 
 Installation instructions for these tools differ heavily based on your operating system and aren't covered here.
 
+### CouchDB on Docker
+
+We recommend using Docker to install and use CouchDB. This ensures you are getting a compatible version and are not relying on OS packages that may be more or less up to date than needed. 
+
+After installing docker, you can create a docker container like so:
+
+```sh
+docker run -d -p 4984:5984 -p 4986:5986 --name medic-couchdb -v /home/scdf/dockers/e2e-couchdb/opt/couchdb/data:/opt/couchdb/data apache/couchdb:2
```

Comment:
This gave me a docker instance with an IP  of `172.17.0.2`.  I'm a total docker newb, but I assume this is NATed and the normal docker behavior?  Do we want to have the happy/default path use these 172.x.x.x IPs such that `COUCH_URL` and `COUCH_NODE_NAME` should be updated to denote which IP you should use (ie not `localhost`)?  Or maybe we feel "Medic recommends you familiarise yourself with other Docker" covers this enough and devs will figure it out?

Reply:
You should definitely be able to use localhost. How did you determine that this was the IP of the docker container?

Reply:
Yup, you're totally right! I didn't fully grok how the port mapping was working (` -p 4984:5984`) in docker, so instead of using `localhost:4984` I found the instance IP with this:

```
docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <instance_name>
```

Having done all the testing and seeing your newest commits with 1:1 port mapping, I now much better understand how this works.  Sorry if it was a distraction!